app/config/collections.php

```diff
@@ -1698,6 +1698,39 @@
             ],
         ],
     ],
+    Database::SYSTEM_COLLECTION_REALTIME_CONNECTIONS => [
+        '$collection' => Database::SYSTEM_COLLECTION_COLLECTIONS,
+        '$id' => Database::SYSTEM_COLLECTION_REALTIME_CONNECTIONS,
+        '$permissions' => ['read' => ['*']],
+        'name' => 'Realtime Connections',
+        'structure' => true,
+        'rules' => [
+            [
+                '$collection' => Database::SYSTEM_COLLECTION_REALTIME_CONNECTIONS,
+                'label' => 'Container',
+                'key' => 'container',
+                'type' => Database::SYSTEM_VAR_TYPE_TEXT,
+                'required' => true,
+                'array' => false,
+            ],
+            [
+                '$collection' => Database::SYSTEM_COLLECTION_REALTIME_CONNECTIONS,
+                'label' => 'Timestamp',
+                'key' => 'timestamp',
+                'type' => Database::SYSTEM_VAR_TYPE_NUMERIC,
+                'required' => true,
+                'array' => false,
+            ],
+            [
+                '$collection' => Database::SYSTEM_COLLECTION_REALTIME_CONNECTIONS,
+                'label' => 'Data',
```

Comment:
data or value? what would make more sense?

Reply:
Actually both, but now you say it I would prefer `value` more.